src/azure-cli/azure/cli/command_modules/appservice/static_sites.py

```diff
@@ -364,8 +364,8 @@ def update_staticsite_users(cmd, name, roles, authentication_provider=None, user
                                           static_site_user_envelope=user_envelope)
 
 
-def create_staticsites(cmd, resource_group_name, name, location,  # pylint: disable=too-many-locals,
-                       source, branch, token=None,
+def create_staticsites(cmd, resource_group_name, name, location="centralus",  # pylint: disable=too-many-locals,
```

Comment:
Did we want to make location optional & Default to CentralUs?

Reply:
> Did we want to make location optional & Default to CentralUs?

Hi [StrawnSC](https://github.com/StrawnSC),
Can you confirm with this ?

Reply:
Yes, this is what @simonaco put on the ADO work item